.github/workflows/build.yml

```diff
@@ -145,11 +149,8 @@ jobs:
       env:
         CTEST_OUTPUT_ON_FAILURE: ON
       run: |
-        ccache --max-size=150M
-        DIR_BUILD="build/ci/release"
-        DIR_SRC="`pwd`"
-        mkdir -p "${DIR_BUILD}" && cd "${DIR_BUILD}"
-        cmake -S "${DIR_SRC}" -D ARCH="default" -D BUILD_SHARED_LIBS=ON -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=release && make -j3 && make test
+        ${{env.BUILD_DEFAULT_LINUX}}
+        make test
```

Comment:
```
ccache --max-size=150M
cmake -S . -B build -D ARCH="default" -D BUILD_TESTS=ON -D CMAKE_BUILD_TYPE=release && make -j3 && make test
```

Reply:
Edit, this might not work because of the existing makefile. Will update my comment.

Reply:
I can try out whatever you feel looks better.

Reply:
My updated comment does what we want in a single line.

Reply:
What about the repetition in line 152?
In order to be sure, that the ccache is reused, this has to be exactly the same chain of cmake commands.

Reply:
Ok, I see your point now. I think this adds quite a bit of complexity just to reuse a single cache.